src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/user/AddOrAmendClaimDocumentsCallbackHandlerTest.java

```diff
@@ -68,6 +69,19 @@ void shouldReturnNoErrors_whenParticularOfClaimsFieldsAreValid() {
             assertThat(response.getErrors()).isEmpty();
         }
 
+        @Test
+        void shouldReturnError_whenParticularOfClaimsTextAndDocumentSubmitted() {
+            CaseData caseData = caseDataBuilder.servedDocumentFiles(ServedDocumentFiles.builder()
+                                                                        .particularsOfClaimText("Some string")
+                                                                        .particularsOfClaimDocument(Document.builder()
+                                                                        .build()).build()).build();
+            CallbackParams params = callbackParamsOf(caseData, MID, pageId);
+
+            var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+            assertThat(response.getErrors()).containsOnly("More than one Particulars of claim details added");
+        }
+
```

Comment:
should there be a test in this file for when just one document added for particulars of claim? i.e doc or text